main/main.c

```diff
@@ -295,7 +295,7 @@ static void init_csp() {
     csp_conf.conn_max = 20;
     csp_conf.conn_queue_length = 10;
     csp_conf.fifo_length = 25;
-    csp_conf.port_max_bind = 24;
+    csp_conf.port_max_bind = 254;
```

Comment:
What does this change do?

Reply:
We ran out of ports for our services so I increased it to the maximum